### PR TITLE
add comments

### PR DESCRIPTION
**Pull request by Gergalov**

### CODE REVIEW

File: src/classes/comment/CommentRepository.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	async addComment(comment) {
+		return  (await axios.post('http://localhost:3000/comments', comment, {
+			headers: {
+				authorization: localStorage.getItem('token')
+			}
+		})).data;
+	}
+
+	getCommentsByUser(id) {
+		return axios.get('http://localhost:3000/comments?author=' + id).then(response => response.data);
```
Comment (untael) [inline src/classes/comment/CommentRepository.js]: Why .then again here? Use async await please

File: src/classes/message/MessageRepository.js [hunk 2]
```diff
@@ -1,18 +1,48 @@
...
 			headers: {
 				authorization: localStorage.getItem('token')
 			}
 		})).data
-		return messages
 	}
 
-	getMessagesByUser(id) {
-		return axios.get('http://localhost:3000/messages?author=' + id).then(response => response.data);
+	createMessage(message) {
```
Comment (untael) [inline src/classes/message/MessageRepository.js]: Why this is not async method? What user can do, if status 500?

File: src/classes/user/UserRepository.js [hunk 3]
```diff
@@ -1,23 +1,46 @@
...
 
-	async signIn(credential) {
-		const token = (await axios.post('http://localhost:3000/auth/login', {credential})).data.access_token
-		return token
+	async findUsers(nick_name) {
+		const user = (await axios.get('http://localhost:3000/users?nick_name_like=' + nick_name)).data;
+		return user.map(UserMapper.map);
+	}
+
+	createUser(user) {
```
Comment (untael) [inline src/classes/user/UserRepository.js]: Not async again, what if server down or some logic broken?

File: src/components/authentication/LoginPage.vue [hunk 4]
```diff
@@ -1,30 +1,62 @@
...
-		<div class="login-page__spacer"/>
+		<div class="login-page__alert">
+			<v-alert
+				:value="true"
+				v-if="status.error"
+				color="error"
+				icon="warning"
+				outline
+			>
+				Wrong email or password!
```
Comment (untael) [inline src/components/authentication/LoginPage.vue:26]: You can get this message from response

File: src/components/authentication/RegisterPage.vue [hunk 5]
```diff
@@ -3,70 +3,100 @@
...
 		name: 'RegisterPagePage',
 		components: {},
 		data() {
 			return {
-				credential: {
-					name: '',
-					password: ''
-				},
-				confirm_password: ''
+				user:  new User,
```
Comment (untael) [inline src/components/authentication/RegisterPage.vue:57]: user : {
type: User,
}
?

File: src/components/layout/AddButton.vue [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+			create
+		</i>
+	</v-btn>
+</template>
+
+<script>
+	export default {
+		name: "AddButton",
+		methods: {
+			createNewMessage() {
```
Comment (untael) [inline src/components/layout/AddButton.vue:30]: Why simple reusable button have any methods? U need to add this method to place where you use it

File: src/components/personal/PersonalPage.vue [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+			this.getUser()
+		},
+		methods: {
+			async getUser() {
+				const id = this.$route.params.user_id;
+				const userRepository = new UserRepository();
+				this.user = await (userRepository.getUserById(id))
+			},
+			async changePassword() {
+				if (this.old_password === this.old_password) {
```
Comment (untael) [inline src/components/personal/PersonalPage.vue]: if (this.old_password === this.old_password) what?

Comment (untael) [reply src/components/personal/PersonalPage.vue]: It always true?

File: src/classes/comment/CommentRepository.js (hunk 1 above)
Comment (Gergalov) [reply src/classes/comment/CommentRepository.js]: fixed

File: src/classes/message/MessageRepository.js (hunk 2 above)
Comment (Gergalov) [reply src/classes/message/MessageRepository.js]: fixed method, add issue "to check user status"

File: src/classes/user/UserRepository.js (hunk 3 above)
Comment (Gergalov) [reply src/classes/user/UserRepository.js]: add 'async', add new task for checking  response

File: src/components/authentication/LoginPage.vue (hunk 4 above)
Comment (Gergalov) [reply src/components/authentication/LoginPage.vue:26]: yes but no, now a can't get response from incorrect get-requests. need to make a catch for that

File: src/components/authentication/RegisterPage.vue (hunk 5 above)
Comment (Gergalov) [reply src/components/authentication/RegisterPage.vue:57]: ok, i done that only for good view that objects in db.json (with all fields)

File: src/components/layout/AddButton.vue (hunk 6 above)
Comment (Gergalov) [reply src/components/layout/AddButton.vue:30]: i don't understand what you mean

File: src/components/personal/PersonalPage.vue (hunk 7 above)
Comment (Gergalov) [reply src/components/personal/PersonalPage.vue]: fixed

**Pull request merged**